components/esp_modem/src/esp_modem_command_library.cpp

```diff
@@ -261,12 +261,17 @@ command_result set_echo(CommandableIf *t, bool on)
     return generic_command_common(t, "ATE0\r");
 }
 
-command_result set_pdp_context(CommandableIf *t, PdpContext &pdp)
```

Comment:
Consider adding the timeout as a default parameter.

Reply:
I'd prefer to keep the two prototypes separately, since this API is one of the generic one, declared here:

https://github.com/espressif/esp-protocols/blob/82c2cf8936d4d363617825c3ed4028d1b0d20386/components/esp_modem/include/generate/esp_modem_command_declare.inc#L98-L102

from which we generate C-API and C++ API (plus others, like docs and DCE's methods)